### DISCUSSION TITLE
Changing information about a Series

### DISCUSSION THREAD

**Discussion by andypiper** [forem/forem#16647] [Q&A] [answered]
When I first created a series, I did not realise that the “name” / label would become the way that it appeared throughout Forem / DEV.

Here’s the example: https://dev.to/andypiper/series

In this case, I’ve made a mistake in the title (should be `esp32c3`, not `esp32c2`, typo); and I didn’t fully understand that this would become the name.

Is there a way to edit this? The series has a numeric ID, so in theory it’s just a value in a DB that I could edit, but I don’t find it in the UI? Also, is there a way to add metadata to the series, for example, OpenGraph tags for the series landing page? At the moment it shares a DEV logo, and the title slug that I’ve mistyped, so it would be cool to be able to update that.

**Comment by ellativity** [comment, marked as answer]
Hey @andypiper I believe there's currently no way for a user to edit their series after it's live. Since this is regarding a Forem-hosted Forem, please email the details of your request to yo@dev.to and we'll get that fixed for you.

This is obviously a bigger issue that won't affect all Forems equally, depending on their comfort in the console. We've had several request for being able to edit series after publication, as summarized by @cmgorton here: https://github.com/forem/forem/discussions/15673 (additional requests and context from @michael-tharrington here: https://github.com/forem/forem/discussions/15320), so thank you for adding your voice to the conversation 🙏

**Reply by ellativity** [reply]
Unmarking this as an answer so it can get more love 😉

**Reply by JennieOcken** [reply]
Agree with Ella, this is the answer right now.  I am actually going to mark that as an answer as I am going to be grouping all these requests together for a future series focused project
